api/routes.ts

```diff
@@ -25,10 +27,14 @@ export async function setup(app:exp.Express) {
     //----------------------------------------------------------
     // Create Services
     //----------------------------------------------------------
-    let idsvc: id.IdentityService = new id.IdentityService();
+    const idDb = new nedb();
+    const idStore = new sm.Store(idDb);
+    let idsvc: id.IdentityService = new id.IdentityService(idStore);
     await idsvc.initialize();
 
-    let qsvc: qm.QuotesService = new qm.QuotesService();
+    const quoteDb = new nedb();
+    const quoteStore = new sm.Store(quoteDb);
```

Comment:
for large apps where this gets un-wieldy, you can use a DI container to automatically wire up all the deps.